controller/controller.go

```diff
@@ -83,6 +83,11 @@ func main() {
 		}
 	}()
 
+	// Listen for database migration, reset connpool on new migration
+	go func() {
+		postgres.ResetOnMigration(db, logger, doneCh)
```

Comment:
No need to wrap in an anonymous function, you can just do `go postgres.ResetOnMigration...`